feature/client-charge/src/commonMain/kotlin/org/mifos/mobile/feature/charge/charges/ClientChargeScreen.kt

```diff
@@ -206,11 +284,339 @@ private fun ClientChargeDialogs(
+            AccountTypeSection(
+                selectedTabLabel = selectedTabLabel,
+                onTabSelected = { newTab ->
+                    selectedTabLabel = newTab
+                    selectedAccountObject = null
+                },
+            )
+
+            if (currentAccountList.isNotEmpty()) {
+                AccountDropdownSection(
+                    label = selectedTabLabel,
+                    accounts = currentAccountList,
+                    selectedAccount = selectedAccountObject,
+                    onAccountSelected = { selectedAccountObject = it },
+                )
+            }
+        }
+
+        ChargeStatusSection(
+            selectedFilter = selectedFilter,
+            onFilterSelected = { selectedFilter = it },
+        )
+
+        Spacer(modifier = Modifier.height(40.dp))
+
+        FilterApplyButton(
+            onClick = {
+                val targetType = when (selectedTabLabel) {
+                    "Savings" -> ChargeType.SAVINGS
+                    "Loan" -> ChargeType.LOAN
+                    "Shares" -> ChargeType.SHARE
+                    else -> ChargeType.CLIENT
+                }
+                onApply(selectedAccountObject, targetType, selectedFilter)
+            },
+        )
+        Spacer(modifier = Modifier.height(24.dp))
+    }
+}
+
+@Composable
+private fun FilterHeader(onClear: () -> Unit) {
+    Row(
+        modifier = Modifier.fillMaxWidth(),
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalAlignment = Alignment.CenterVertically,
+    ) {
+        Text(
+            text = "Filter Charges",
+            style = MaterialTheme.typography.titleLarge.copy(
+                fontWeight = FontWeight.Bold,
+                fontSize = 20.sp,
+            ),
+        )
+        TextButton(onClick = onClear) {
+            Text(
+                text = "Clear All",
+                style = MaterialTheme.typography.bodyMedium.copy(
+                    color = MaterialTheme.colorScheme.primary,
+                    fontWeight = FontWeight.SemiBold,
+                ),
+            )
+        }
+    }
+}
+
+@Composable
+private fun AccountTypeSection(
+    selectedTabLabel: String,
+    onTabSelected: (String) -> Unit,
+) {
+    Column {
+        Text(
+            text = "Account Type:",
+            style = MaterialTheme.typography.bodyMedium.copy(fontWeight = FontWeight.Medium),
+            modifier = Modifier.padding(vertical = 8.dp),
+        )
+        Row(horizontalArrangement = Arrangement.spacedBy(12.dp)) {
+            val types = listOf("Savings", "Loan", "Shares")
+            types.forEach { type ->
+                FilterOptionChip(
+                    label = type,
+                    isSelected = selectedTabLabel == type,
+                    onClick = { onTabSelected(type) },
+                    modifier = Modifier.weight(1f),
+                )
+            }
+        }
+        Spacer(modifier = Modifier.height(16.dp))
```

Comment:
Same here, please review.
```suggestion
        Spacer(modifier = Modifier.height(KptTheme.spacing.md))
```